### ISSUE TITLE
Batch Chain Events Notifications

### ISSUE THREAD

**Issue by jnaviask** [hicommonwealth/commonwealth#2462]
Spec: https://www.notion.so/hicommonwealth/Notifications-Refactor-9869425462db405c8c3f2f3b28ed080e

**Comment by CowMuon** [comment]
When should we expect details & tag to be filled out? @jnaviask @timolegros

**Comment by jnaviask** [comment]
Tim to fill out details.

**Comment by timolegros** [comment]
This issue is irrelevant for the moment. The Notifications service should be done first for 2 reasons:
1. Refactoring notifications as a whole is a higher priority. We are currently running into scalability issues and even though we have capabilities for improved notifications on the client it currently isn't possible to implement these improvements with the notifications system we have. Not to mention that 99% of all chain-event notifications prior to the CET removal PR that is to be merged soon were unused.
2. If we implement batching notifications first we will have to re-implement it all over again for the notifications service that comes right after.

For the time being I have added the 'wontfix' label to indicate there is no on-going work for this issue.

**Comment by timolegros** [comment]
Closing as this will all be reworked/re-designed for the notifications service.
